test_sd_whonix_config AssertionError: 17 != '17'

* [ ] I have searched for duplicates or related issues

## Description

Via https://github.com/freedomofpress/securedrop-workstation/pull/1295#issuecomment-2863526309

```
INFO:[2025-05-08-15:30:25:930148] ======================================================================
INFO:[2025-05-08-15:30:25:930195] FAIL: test_sd_whonix_config (test_vms_exist.SD_VM_Tests.test_sd_whonix_config)
INFO:[2025-05-08-15:30:25:930240] ----------------------------------------------------------------------
INFO:[2025-05-08-15:30:25:930287] Traceback (most recent call last):
INFO:[2025-05-08-15:30:25:930333]   File "/home/user/securedrop-workstation/tests/test_vms_exist.py", line 107, in test_sd_whonix_config
INFO:[2025-05-08-15:30:25:930382]     self.assertEqual(
INFO:[2025-05-08-15:30:25:930441] AssertionError: 17 != '17'
```

--- a/tests/test_vms_exist.py
+++ b/tests/test_vms_exist.py
@@ -106,9 +106,9 @@ class SD_VM_Tests(unittest.TestCase):
         # Whonix version checking
         self.assertEqual(
             Util.get_whonix_version(),  # If mismatch, whonix may have been updated.
-            CURRENT_WHONIX_VERSION,  # Fix the test by bumping CURRENT_WHONIX_VERSION
+            int(CURRENT_WHONIX_VERSION),  # Fix the test by bumping CURRENT_WHONIX_VERSION
         )
-        self.assertEqual(vm.template.features.get("os-version"), int(CURRENT_WHONIX_VERSION))
+        self.assertEqual(vm.template.features.get("os-version"), CURRENT_WHONIX_VERSION)
 
     def test_sd_proxy_config(self):
         vm = self.app.domains["sd-proxy"]

